index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Capstone Project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script defer src="script.js"></script>
+</head>
+<body>
+    <header> 
+        <ul class="nav">
+            <li><a href="#"><img src="images/facebook.png" alt="facebook logo" width="25px" height="25px"></a></li>
+            <li><a href="#"><img src="images/twitter.png" alt="twitter logo" width="25px" height="25px"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+        <div class="nav-container">
```

Comment:
Kindly use semantic `nav` element instead of `div`element to make your web page SEO friendly.